portable_fs/template/fs.py

```diff
@@ -11,6 +11,7 @@
 from . import core
 from collections import defaultdict
 import warnings
+import pathlib
```

Comment:
This is Python 3 only. Need to add pathlib backport package as a Python 2 dependency.